### PR TITLE
Add Room Dependency

### PR DESCRIPTION
**Pull request by jmseb3**
1.append Room Dependency 
2.Delete ComposeOption
(kotlin2.0.0 above not need)

### CODE REVIEW

File: src/commonMain/kotlin/wizard/files/composeApp/ModuleBuildGradleKts.kt [hunk 1]
```diff
@@ -11,12 +11,17 @@ class ModuleBuildGradleKts(info: ProjectInfo) : ProjectFile {
...
                 dep.isPlugin() -> plugins.add(dep)
                 dep.isCommon() -> {
-                    if (!dep.isTestDependency) commonDeps.add(dep)
-                    else commonTestDeps.add(dep)
+                    if (dep.isKspDependency) {
+                        kspDeps.add(dep)
+                    } else {
+                        if (!dep.isTestDependency) commonDeps.add(dep)
+                        else commonTestDeps.add(dep)
+                    }
```
Comment (terrakok) [inline src/commonMain/kotlin/wizard/files/composeApp/ModuleBuildGradleKts.kt:24]: `isKspDependency` is a top level characteristic

File: src/commonMain/kotlin/wizard/files/composeApp/ModuleBuildGradleKts.kt [hunk 2]
```diff
@@ -11,9 +11,17 @@ class ModuleBuildGradleKts(info: ProjectInfo) : ProjectFile {
         val otherDeps = mutableSetOf<Dependency>()
         val commonTestDeps = mutableSetOf<Dependency>()
         val otherTestDeps = mutableSetOf<Dependency>()
+        val kspDeps = mutableSetOf<Dependency>()
+        val kspTestDeps = mutableSetOf<Dependency>()
```
Comment (terrakok) [inline src/commonMain/kotlin/wizard/files/composeApp/ModuleBuildGradleKts.kt:15]: only kspDeps is enough

Comment (terrakok) [reply src/commonMain/kotlin/wizard/files/composeApp/ModuleBuildGradleKts.kt:15]: do you know test ksp libraries? let's keep it simple for now

**Review by terrakok — CHANGES_REQUESTED** [review]

File: src/commonMain/kotlin/wizard/files/composeApp/ModuleBuildGradleKts.kt (hunk 2 above)
Comment (jmseb3) [reply src/commonMain/kotlin/wizard/files/composeApp/ModuleBuildGradleKts.kt:15]: All right. I will work without test

**Comment by jmseb3** [comment]
on ios only Project 
not work ksp :(

~~~
Cannot change attributes of configuration ':composeApp:debugFrameworkIosX64' after it has been locked for mutation
~~~

File: src/commonMain/kotlin/wizard/files/composeApp/ModuleBuildGradleKts.kt [hunk 3]
```diff
@@ -279,5 +292,53 @@ class ModuleBuildGradleKts(info: ProjectInfo) : ProjectFile {
...
+                    if (info.hasPlatform(ProjectPlatform.Mingw) && dep.hasPlatform(ProjectPlatform.Mingw)) {
+                        appendLine("$addDepth        add(\"kspMingwX64\", this)")
+                    }
+                    appendLine("$addDepth    }")
+                }
+                appendLine("$addDepth}")
+            }
+            val isIOSOnly = info.platforms == setOf(ProjectPlatform.Ios)
+            if (isIOSOnly) {
+                appendLine("afterEvaluate {")
```
Comment (terrakok) [inline src/commonMain/kotlin/wizard/files/composeApp/ModuleBuildGradleKts.kt:336]: If you use a hack in your scripts you have to add a detailed description about the reason

**Comment by jmseb3** [comment]
It was a really good experience.
thank you! :)

**Pull request merged**